jcstress-core/src/main/java/org/openjdk/jcstress/infra/runners/TestConfig.java

```diff
@@ -244,4 +244,23 @@ public void generateDirectives(PrintWriter pw, Verbosity verbosity) {
         pw.println("]");
         pw.flush();
     }
+
+
+    public String toDetailedTest() {
+        //binaryName have correct $ instead of . in name; omitted
+        //generatedRunnerName name with suffix (usually _Test_jcstress) omitted
+        //super.toString() as TestConfig@hash - omitted
+        return name +
+                " {" + actorNames +
+                ", spinLoopStyle: " + spinLoopStyle +
+                ", threads: " + threads +
+                ", forkId: " + forkId +
+                ", maxFootprintMB: " + maxFootprintMB +
+                ", compileMode: " + compileMode +
+                ", shClass: " + shClass +
+                ", strideSize: " + strideSize +
+                ", strideCount: " + strideCount +
+                ", cpuMap: " + cpuMap +
+                ", " + jvmArgs + "}";
```

Comment:
kindly modify this to return a valid json, perhaps try running this method locally once in your machine, and copy paste that json in any online json formatter to check if you encounter any errors

Reply:
I never had an intention to return valid json.  It is used only in verbose output, to differentiate individual variants of tests.  Wihtout the test id, it do no have sense, and the extended info in {} is... complicated. See eg:
```
org.openjdk.jcstress.samples.api.API_01_Simple {[actor1, actor2], spinLoopStyle: Thread.onSpinWait(), threads: 2, forkId: 0, maxFootprintMB: 64, compileMode: 0, shClass: (PG 0, CG 0), (PG 0, CG 0), strideSize: 256, strideCount: 40, cpuMap: null, [-XX:+UseBiasedLocking]}
org.openjdk.jcstress.samples.api.API_01_Simple {[actor1, actor2], spinLoopStyle: Thread.onSpinWait(), threads: 2, forkId: 0, maxFootprintMB: 64, compileMode: 0, shClass: (PG 0, CG 0), (PG 0, CG 0), strideSize: 256, strideCount: 40, cpuMap: null, [-XX:-UseBiasedLocking]}

```

To highlight it is not json, was the original usage of `=`,  but I did not had super strong opinion.
As the output should remain mainly human readable, I'm quite against full json (thus bracket before name, and id before actorNames and jvmArgs).  But on contrary,  the list is indeed huge.

The main motivation is the line of ""All matching tests combinations - " + testsToPrint.size()" thus real number of run tests. The listing itself is (very) useful candy on top:
```
java -jar tests-all/target/jcstress.jar -v -l  -t org.openjdk.jcstress.samples.api.API_01_Simple 
...
All matching tests combinations - 96
...
```
x
```
java -jar tests-all/target/jcstress.jar -l  -t org.openjdk.jcstress.samples.api.API_01_Simple 
...
All matching tests - 1
...
```


So back to verbose listig output format - I intentionally kept it semi parseabel - grep/sed  on output, and readable enough: name  info in {} , all firelds with key id, and obvious arrays in [] without key.
AFAICJ, the full json will lower the human readabiity. 
What about this - with "-v" it would print it as it is - human readable, fully distinguishible.  on "-vv or -vvv"  it would print complete fully qulified json. WDYT?

Reply:
ok sure, gotcha, the human readable output with `-v` option what you are printing right now should be fine, just one small change I would request, instead of doing string concatenation with the `+` operator, can we use append  operation with String Builder or String Buffer whichever is more appropriate here

Reply:
Thanx, done!
Maybe.. can I return back the "=" instead of ": " ?

Reply:
I'd suggest to probably let it be, because the name is there at the start following the convention of other tests in the `else` clause where just the name is printed. Here in the `if` clause we have the additional content wrapped in `{  }` , although its partial json, it can still be modified to complete json with less changes in future if required

Reply:
Yah, I thought the same:) Ok by me. TYVM!